frontend/src/components/WindSettings.jsx

```diff
@@ -20,391 +19,209 @@ const useStyles = makeStyles((theme) => ({
-                                    }
-                                }
+        <Grid container spacing={5} direction="column" classes={{ root: classes.transparentBackground }}>
+            <Grid item>
+                <FormControlLabel
+                    control={
+                        <Switch
+                            checked={isWindSelectionEnabled}
+                            onChange={(e) => setIsWindSelectionEnabled(e.target.checked)}
+                            color="primary"
+                            sx={{
+                                '& .MuiSwitch-track': {
+                                    backgroundColor: '#F5F5DC',
+                                },
+                                '& .MuiSwitch-thumb': {
+                                    color: '#FFFFFF',
+                                },
                             }}
-                            onChange={handleWindTypeChange}
-                            fullWidth
-                            >
-                            {WindType.map((val) => (
-                                <MenuItem value={val.value} key={val.id}>
-                                    <em>{val.value}</em>
-                                </MenuItem>
-                            ))}
-                        </StyledSelect>
-                    </Grid>
-                </Grid>
+                        />
+                    }
+                    label={<span style={{ color: '#FFFFFF' }}>Enable Wind Selection</span>}
+                />
+            </Grid>
 
-                <Grid item container alignItems="center" direction="row">
-                    <Grid item xs={4}>
-                        <InputLabel id="direction-label" sx={{ marginRight: 2, flexShrink: 0, color: '#F5F5DC', width: '200px' }}>
-                            Wind Direction
-                        </InputLabel>
-                    </Grid>
-                    <Grid item xs={6}>
-                        <StyledSelect
-                            labelId="direction-label"
-                            input={<OutlinedInput/>}
-                            MenuProps= {{
-                                sx: {
-                                    '& .MuiPaper-root': {
-                                        backgroundColor: '#F5F5DC',
-                                    }
-                                }
-                            }}
-                            value={envConf.Wind.Direction}
-                            onChange={handleDirection}
-                            fullWidth
-                            >
-                            {WindDirection.map((val) => (
-                            <MenuItem value={val.value} key={val.id}>
-                                <em>{val.value}</em>
-                            </MenuItem>
-                            ))}
-                        </StyledSelect>
+            {!isWindSelectionEnabled && (
+                <Grid item container xs={12}>
```

Comment:
@syedomair0 remove the condition `!isWindSelectionEnabled` and move this Grid to new line [210](!isWindSelectionEnabled), as we wanted to have "+ Add Wind Source" button at the bottom of all the blocks.